api/index.ts

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+
+
+const app = express();
+
+const events=require("./eventsInput");
+const array = [events];
+
+
+app.get('/api', (req, res) => res.send(array));
+
+app.listen(4000, () => {
```

Comment:
Let's change the port to 5080 to distinguish it from the client's port group

Reply:
Also, remove node dependencies from the package.json in client-app and fix the formatting